pkgs/tools/audio/mpris-scrobbler/default.nix

```diff
@@ -1,5 +1,7 @@
-{ lib, stdenv
+{ lib
+, stdenv
 , fetchFromGitHub
+, fetchpatch
```

Comment:
`fetchpatch` isn't used, did you want to apply upstream patches for the `format-truncation` and `strinop-overflow` errors?

Reply:
Yes, but that did not help much. Removed.